.goreleaser.yml

```
@@ -9,7 +9,7 @@ before:
     - go test ./...
     # As part of the release doc files are included as a separate deliverable for
     # consumption by Packer.io. To include a separate docs.zip uncomment the following command.
-    #- /bin/sh -c "[ -d docs ] && zip -r docs.zip docs/"
+    - /bin/sh -c "[ -d docs ] && zip -r docs.zip docs/"
```

Fixing this to render partials
```suggestion
    - make ci-release-docs
```